src/System.IO.Pipes.AccessControl/src/System.IO.Pipes.AccessControl.csproj

```diff
@@ -2,29 +2,11 @@
   <PropertyGroup>
     <AssemblyName>System.IO.Pipes.AccessControl</AssemblyName>
     <ProjectGuid>{40059634-BB03-4A6F-8657-CCE2D376BC8B}</ProjectGuid>
-    <IncludeDefaultReferences Condition="'$(TargetGroup)' == 'netcoreapp' OR '$(TargetGroup)' == 'uap'">false</IncludeDefaultReferences>
-    <AllowUnsafeBlocks Condition="'$(TargetGroup)' == 'netstandard'">true</AllowUnsafeBlocks>
-    <GeneratePlatformNotSupportedAssemblyMessage Condition="'$(TargetGroup)' == 'netstandard' AND '$(TargetsWindows)' != 'true'">SR.PlatformNotSupported_AccessControl</GeneratePlatformNotSupportedAssemblyMessage>
-    <IsPartialFacadeAssembly Condition="'$(TargetGroup)' != 'netstandard'">true</IsPartialFacadeAssembly>
+    <IncludeDefaultReferences>false</IncludeDefaultReferences>
+    <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
     <Configurations>net461-Windows_NT-Debug;net461-Windows_NT-Release;netcoreapp-Windows_NT-Debug;netcoreapp-Windows_NT-Release;netfx-Windows_NT-Debug;netfx-Windows_NT-Release;netstandard-Debug;netstandard-Release;netstandard-Windows_NT-Debug;netstandard-Windows_NT-Release;uap-Windows_NT-Debug;uap-Windows_NT-Release</Configurations>
```

Comment:
sync Configurations with Configurations.props

Reply:
I quickly spoke with @eerhardt. He mentioned that you were trying to combine the Configurations.props with the Configurations prop here. Do we have a tracking issue for that?

Reply:
Thanks for the reminder as I planned to do that all at once for all the projects.

Reply:
>I quickly spoke with @eerhardt. He mentioned that you were trying to combine the Configurations.props with the Configurations prop here. Do we have a tracking issue for that?

Not there isn't a tracking issue but I did make some attempts to figure out a good way to take the configurations in configurations.props and add the "Debug" and "Release" to them but my attempts have failed. So for now I think we will stick with the update task but I may make another attempt once we get our configuration system ported to arcade.

Reply:
ok thanks for the update